drivers/bridge/bridge.go

```diff
@@ -466,9 +474,7 @@ func parseNetworkGenericOptions(data interface{}) (*networkConfiguration, error)
 		config = opt
 	case map[string]interface{}:
 		config = &networkConfiguration{
-			EnableICC:          true,
-			EnableIPTables:     true,
-			EnableIPMasquerade: true,
```

Comment:
@mbanikazemi Please fix this ^ in your next push, we just realized this comment was not taken care of. This explains why the ip masquerade flag is not true for other than docker0 bridge networks.